kafka/src/main/java/org/axonframework/extensions/kafka/eventhandling/consumer/OffsetCommitType.java

```diff
@@ -0,0 +1,10 @@
+package org.axonframework.extensions.kafka.eventhandling.consumer;
```

Comment:
New files should have a copyright notice to state all is Apache 2 licensed. You can find the copyright notice template [here](https://github.com/AxonFramework/AxonFramework/blob/master/copyright-template.xml). Or, you can copy it from another file in this repository and adjust the date accordingly.